src/sd_api_v2/sdk/components/serialization/application/codecs/common/conn_systemreset_app.h

```diff
@@ -66,7 +77,8 @@ extern "C" {
  * @retval NRF_SUCCESS          Encoding success.
  * @retval NRF_ERROR_INTERNAL   Encoding failure. Transport error.
  */
-uint32_t conn_systemreset(void);
+uint32_t conn_systemreset(uint8_t * const  p_buf,
```

Comment:
Since this is the encode function it should be named conn_systemreset_enc.

Reply:
Agree